metrics/ap_association.yaml

```diff
@@ -55,6 +55,13 @@ PackageAlertsCpuTime:
     <<: *CpuTime
     description: CPU time elapsed in PackageAlertsTask.
 
+TransformDiaSourceCatalogTime:
+    <<: *RunningTime
+    description: Wall-clock time elapsed in TransformDiaSourceCatalogTask.
+TransformDiaSourceCatalogCpuTime:
+    <<: *CpuTime
+    description: CPU time elapsed in TransformDiaSourceCatalogTask.
+
```

Comment:
Oops, spoke too soon. We already have these metrics as `MapDiaSourceTime` (`MapDiaSourceTask` was the original name for the same task). I suppose we should stick to the new name just to keep anybody from making the same mistake in the future?

Reply:
(For now I've taken the liberty of adding the old metric to the dashboard, since even if we do switch names we'd still want to display the old data.)

Reply:
Let's keep the old one. I might reword "MapDiaSourceTask or TransformDiaSourceCatalogTask" (line 35) to something like "TransformDiaSourceCatalogTask, formerly called MapDiaSourceTask" to make it more clear they're the same thing.